docs/api/browser-window.md

```diff
@@ -219,6 +219,23 @@ Emitted when devtools is closed.
 
 Emitted when devtools is focused / opened.
 
+### Event: 'app-command' (Windows):
```

Comment:
We currently say `__Note__: This event is available only on Windows.` in event's description instead of putting `(Windows):` in the title.